src/packer.pkr.hcl

```
@@ -1,3 +1,16 @@
+variable "winrm_password" {
+  default     = false
```

So we match the type of the variable. Please also add an issue and comment mentioning that this only has a default because of a current limitation with the [pre-commit-packer](https://github.com/cisagov/pre-commit-packer) hook. Ideally we would correctly mark this as a required variable.

```suggestion
  default     = ""
```

---

@mcdonnnj makes sense. resolved with [2973420](https://github.com/cisagov/windows-server-packer/pull/10/commits/29734201bc39508d11674b968a889785ba6b5cb6)

---

@itsmostafa - I see that you created issue #21 for this, however you didn't include a comment and a link to that issue in `packer.pkr.hcl`.  

Also, please flesh out #21 a bit more so that it contains some more useful information and context.  It might make more sense in this case to use our [issue idea template](https://github.com/cisagov/windows-server-packer/issues/new?assignees=&labels=&template=issue.md), rather than the [bug template](https://github.com/cisagov/windows-server-packer/issues/new?assignees=&labels=&template=bug.md) that you chose.

---

@dav3r @mcdonnnj i updated the issue and resolved with [a1dee3d](https://github.com/cisagov/windows-server-packer/pull/10/commits/a1dee3d09766e24affcba2cc164cb21c2dee7d77)

---

@mcdonnnj resolved with [5b7875f](https://github.com/cisagov/windows-server-packer/pull/10/commits/5b7875f21d26a670bb91bff12bbaaff0bc66e2ba)